template/src/features/counter/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
```

Comment:
Oh, one other naming scheme thing.

I'd like to change this to `Counter.js`.

I've never been a fan of the "using `index.js` for all the components based on folders" thing.

Reply:
sure. Should the folder name be `counter` or `Counter` ?

Reply:
`features/counter/Counter.js`, please